,

```diff
@@ -6,6 +6,11 @@
 # If there are multiple candidates, the user chooses one using `fzy`.
 set -euo pipefail
 
+dbOutdatedError(){
+  echo "The database used by comma might be outdate. To update it, run:"
+  echo ", --update"
+}
+
```

Comment:
been using this pr for some time, i don't think we should have this

Reply:
Why not?

Reply:
because if a program has a error then this message will be shown